WalletWasabi.Fluent/ViewModels/Wallets/Send/SuggestionViewModel.cs

```diff
@@ -1,16 +1,17 @@
+using System.Windows.Input;
 using WalletWasabi.Blockchain.TransactionBuilding;
 
 namespace WalletWasabi.Fluent.ViewModels.Wallets.Send;
 
 public class SuggestionViewModel : ViewModelBase
 {
-	public SuggestionViewModel(BuildTransactionResult transactionResult, bool isOriginal = false)
+	public SuggestionViewModel(BuildTransactionResult? transactionResult, bool isOriginal = false)
```

Comment:
`SuggestionViewModel` should be an `abstract` class without any parameter.

`transactionResult` and `isOriginal` should be moved to `ChangeAvoidanceSuggestionViewModel`. They have nothing to do with `PocketSuggestionViewModel`.

Because if I understand correctly the purpose of the `SuggestionViewModel` is to have `ChangeAvoidanceSuggestionViewModel` and `PocketSuggestionViewModel` both in the same `Suggestions` list.

Reply:
very good, fixed.